acsylla/_cython/result/value.pyx

```diff
@@ -17,39 +17,36 @@ cdef class Value:
     def int(self):
         """ Returns the int value of a column.
 
-        Raises a `ColumnValueError` if the value can not be retrieved"""
+        Raises an Exception if the value can not be retrieved"""
```

Comment:
Could we say something like

```
Raises a `CassException` derivated class if the value can not be retrieved
````

The same for other places where. Also, we would need to modify the interfaces documentation if its needed [1]

[1] https://github.com/pfreixes/acsylla/blob/master/acsylla/base.py#L176

Reply:
sure. done.